src/util/Filter.js

```diff
@@ -579,7 +579,7 @@ function filterApply(card, filter) {
     });
   }
   if (filter.category == 'status') {
-    if (card.status.toLowerCase() == filter.arg) {
+    if (card.status == filter.arg) {
```

Comment:
```suggestion
    if (card.status.toLowerCase() == filter.arg.toLowerCase()) {
```
Let's do this instead to prevent any further issues with case sensitivity